### PR TITLE
[echelon] add plugins that echelon depends on

### PR DESCRIPTION
**Pull request by ChrisLundquist**
Completely based on work found in https://github.com/heavywater/echelon
- carbon.rb
- chef.rb
- graphite.rb
- librato.rb
- load.rb
- qmail_queue.rb

### CODE REVIEW

File: recipes/chef.rb [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#
+# Cookbook Name:: collectd_plugins
+# Recipe:: chef
+#
+# Copyright 2010, Atari
+#
+# All rights reserved - Do Not Redistribute
```
Comment (coderanger) [inline recipes/chef.rb:7]: Hah, I should probably update this seeing as I wrote it and clearly this isn't internal anymore (I had it as a gist I sent to a few people, not sure how you got a hold of it :-)

**Comment by ChrisLundquist** [comment]
I don't know the whole story. I just found it in https://github.com/heavywater/echelon and I am trying to make it work with Berkshelf (http://berkshelf.com/) instead of having all the cookbooks being vendor-ed.

**Comment by coderanger** [comment]
If you aren't the copyright owner on this code I'm not sure how I feel about accepting it. Conveniently I'll probably see AJ tomorrow so I can just ask him bun in the future you should really get sign off from the owner for something like this.

**Comment by ChrisLundquist** [comment]
Yeah, I am not trying to pass this off as my own by any means. I am only trying to get the HeavyWater changes merged up stream so I don't have a ghetto berksfile which points to a bunch of custom repos and branches, making it no better than vendoring the cookbooks.

(Apache 2)
https://github.com/heavywater/echelon/blob/master/LICENSE

Is what I was going off of.

Yeah I was supposed to grab a beer with AJ tomorrow, I would love to talk all this out.

**Comment by coderanger** [comment]
Yeah, I'm about 90% sure its actually legal under APL even if he objected, but of course it is AJ so he doesn't care either way :-)

**Comment by ChrisLundquist** [comment]
Tweeted this at you both, because you know, everyone likes working on Sunday nights.

**Comment by ChrisLundquist** [comment]
shall we close or merge this?

**Comment by coderanger** [comment]
So looking at this again, you reference a bunch of cookbook_files and templates that you didn't copy over.

**Comment by ChrisLundquist** [comment]
you're right. Let me close this out.

**Pull request closed without merging**